src/CMakeLists.txt

```diff
@@ -62,6 +62,8 @@ set(USE_GETTEXT FALSE)
 
 if(ENABLE_GETTEXT)
 	find_package(GettextLib)
+	find_package(Iconv)
+	find_package(Intl)
```

Comment:
These are not required for Linux.

Reply:
@SmallJoker Is it worth adding if(CMAKE_SYSTEM_NAME MATCHES "kOpenBSD.*|OpenBSD.*")?

Reply:
@SmallJoker argh, they were required in the past, so ignore this patch.